Explorer/Assets/Scripts/SceneRuntime/WrapBunch.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.ClearScript.V8;
+using System;
+using System.Collections.Generic;
+
+namespace SceneRuntime
+{
+    public class WrapBunch : IDisposable
```

Comment:
let's give it a more specific meaningful name: I believe it's dedicated to APIs only, and we would not like to mix it with anything else.